js/modules/k6/http/http.go

```diff
@@ -47,22 +51,35 @@ var (
 	typeURLTag = reflect.TypeOf(URLTag{})
 )
 
+type OCSPStapledResponse struct {
+	ProducedAt, ThisUpdate, NextUpdate, RevokedAt string
+	RevocationReason                              string
+	Status                                        string
+}
+
+type OCSP struct {
+	StapledResponse OCSPStapledResponse
```

Comment:
Is this indirection really needed...?

Reply:
Probably not, maybe it should just be `res.ocsp_response` on the JS side?

Reply:
Or maybe even just `res.ocsp`?